catalog/app/containers/Bucket/Meta.tsx

```diff
@@ -85,58 +73,78 @@ interface PackageMetaProps extends Partial<SectionProps> {
 function PackageMetaSection({ meta, preferences, ...props }: PackageMetaProps) {
   const classes = usePackageMetaStyles()
   const { message, user_meta: userMeta, workflow } = meta
+  const [expanded, setExpanded] = React.useState({
```

Comment:
combining these states seem like an unnecessary complication -- it would be simpler to just have two separate states

Reply:
Yeah, since I still have two callbacks anyway `onMetaToggle` and `onWorkflowToggle`
I made one callback, but the code looked too complicated for such trivial operation

I'll revert to two states